CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ Note: this log contains only changes from knife-ec2 release 0.8.0 and later
 -- it does not contain the changes from prior releases. To view change history
 prior to release 0.8.0, please visit the [source repository](https://github.com/chef/knife-ec2/commits).
 
+
+## Latest Release 0.15.0 (2017-02-03)
+* `knife-ec2` requires `chef-client 12.14` or newer.
+* [knife-ec2:#471](https://github.com/chef/knife-ec2/pull/471) Added support to append ec2 server id into node name using -N "TestCLI%s" 
+
```

Comment:
The changelog is incomplete. Please add the reference for all the PRs done after release 0.14.0.